### ISSUE TITLE
usage of issues in this github repro for contact

### ISSUE THREAD

**Issue by OwenBrotherwood** [TheThingsNetworkDenmark/TheThingsNetworkDenmark#8]
opinionated:
usage of emails as contact can be a problem
possible solution
use github issues
pro's
gives some form of issue (contact) list 
con's
emails cannot be sent at the spur of the moment (can be  a pro)
no github account: ask a friend to create an issue or just create an account

**Comment by mikini** [comment]
I'm not too keen on using Github Issues for pure communication, that's just abuse and quirky, why are we even doing that when no code is involved?

As far as I have seen the TTN main site has some nice facilities for collaboration and communication, at least for the technical stuff ([wiki](https://www.thethingsnetwork.org/wiki/), [forum](https://www.thethingsnetwork.org/forum), [docs](https://www.thethingsnetwork.org/docs/), [labs](https://www.thethingsnetwork.org/labs/)).

Is something wrong with the [Discourse](http://www.discourse.org/) forum already set up at https://www.thethingsnetwork.org/forum/c/country/denmark?

**Comment by OwenBrotherwood** [comment]
![image](https://cloud.githubusercontent.com/assets/866122/23585001/39c52110-0172-11e7-8298-dfa78c6142a1.png)

All of the above are available, I asked for the forum: people may use it. I have filled in the country TTN template for Denmark with all the information that is required.

If we are sure that people are answering questions from others about TTN in Denmark with these communication channels, then that is ok.

![image](https://cloud.githubusercontent.com/assets/866122/23585062/9e0961bc-0173-11e7-9e2e-09e633623fa4.png)


The question is, at the bottom of the page for country denmark, what is the best way to obtain contact?
The current web page shows a random community in denmark, if the page is refreshed (once or more times) different community people are shown. 
The implementation of the web page is TTN and the way that the communities are scrolled, is also TTN method.

At the current time, I have asked for a denmark@<ttn email> that can be forwarded to more than one person: the issue, or "comment in slack" is not traceable, so I do not know the status.

The usage of forum, with sticky comments to ensure some form of contact can be used: feel free to implement such that there is a contact to denmark.
The slack has sticky comments as well, feel free to implement
Currently, on slack, some people are asking what to use for a gateway: the question can be what is being used in denmark or "hello world" what are you using. What is being done, used and implemented in Denmark could be forum, wiki, slack or github (github gets used for other things than code)

Yes: github to show teams (faces) may be quirky: but it _could_ be used: an opinionated view and the best is to find the correct solution that can be for all (hard but not impossible)

**Comment by OwenBrotherwood** [comment]
You will also find that the TTN global has a lot of work to do.
This can mean that any country efforts that require some form of push, have a slight delay.
For example, Esbjerg community may take a time to appear on country denmark: time will tell.
ie: the point of this github is to allow for country to get going without waiting for TTN global with the issues that are country specific.
Decentralisation of communications but still using TTN global communications where it gives purpose/meaning/ignition

**Comment by OwenBrotherwood** [comment]
Tracking the perceived need for a denmark @ (ttn domain) with forward to more than one person
https://thethingsnetwork.slack.com/archives/initiators/p1488700771000543
lets see if the request gets some traction

Found the original request:
https://thethingsnetwork.slack.com/archives/initiators/p1488291117000355

**Comment by avbentem** [comment]
As TTN's Slack has a limit of 10,000 messages, so purges one whenever a new one is posted, here's the February 28th Slack conversation for your future reference:

> owen_brotherwood [2:57 PM]  
> Is it possible to have an email like denmark@thethingsnetwork.org so that one does not have to post one personnel email address?
I don't mind using my own, but then the contact to people is dependant on me reading my emails (or not missing the email contact)
> With possible forward to more than one receiver ... (edited)

> laurens.slats [3:11 PM]  
> @owen_brotherwood good suggestion, we might want to consider that in a later phase but only if we know for sure that the people behind that mail address know all the ins-and-out of The Things Network

**Comment by OwenBrotherwood** [comment]
yes: slack purge means that slack is for "just now " messages and as such is a communications channel for here and now
